src/PropertiesPanel.js

```diff
@@ -217,6 +217,18 @@ export default function PropertiesPanel(props) {
     errors
   };
 
+  const [ activePopupEntryIds, setActivePopupEntryIds ] = useState([]);
```

Comment:
:spiral_notepad:  We still need to manage the visual statue of fields that are currently popped up in the properties panel. To achieve this, we're using this piece of state and interacting with it via event, similarly to how errors currently work.